collector/README.md

```diff
@@ -61,93 +61,96 @@ marked with a '?' in the `compare` page.
 
 ### How to benchmark a change on your own machine
 
-To benchmark a local build:
+The following command runs the benchmark suite using a local rustc:
 ```
-./target/release/collector --db $DATABASE \
-    bench_local --rustc $RUSTC --cargo $CARGO $ID
+./target/release/collector bench_local <RUSTC> <ID>
 ```
 
-`$DATABASE` is a path (relative or absolute) to a sqlite database file, in which
-the timing data will be placed. It will be created if it does not already exist.
-Alternatively, the collector supports postgres as a backend and the URL can be
-specified (beginning with `postgres://`), but this is unlikely to be useful for
-local collection.
+It will benchmark the entire suite and put the results in a file called
+`results.db`. The full benchmark suite takes some time to run: tens of minutes
+or more, depending on the speed of your machine. Progress output is printed to
+stderr.
 
-`$RUSTC` is a path (relative or absolute) to a rustc executable. Some
-benchmarks use procedural macros, which require a stage 2 compiler. Therefore,
-the value is likely to be something like
-`$RUSTC_REPO/build/x86_64-unknown-linux-gnu/stage2/bin/rustc`, where
-`$RUSTC_REPO` is a path (relative or absolute) to a rustc repository.
+The following arguments are mandatory.
 
-`$CARGO` is a path (relative or absolute) to a Cargo executable. Using an
-installed Cargo is fine, e.g. ``--cargo `which cargo` ``, though you should
-prefer a nightly cargo -- and note that things may not closely match production
-measurements as Cargo sometimes begins passing (or stops passing) various flags
-to rustc.
+- `<RUSTC>`: a path (relative or absolute) to a rustc executable that will be
+  benchmarked. Some benchmarks use procedural macros, which require a stage 2
```

Comment:
A stage 2 compiler shouldn't be required for proc macros. The only case where stage 2 is needed is the script-servo and (maybe) style-servo benchmarks, which use plugins.